commit 977f8fce0279e5f96dc5c5068610d60b9ae94802
Author: Johan Hedberg <johan.hedberg@intel.com>
Date:   Thu Jul 17 15:35:39 2014 +0300

    Bluetooth: Introduce a flag to track who really initiates authentication
    
    Even though our side requests authentication, the original action that
    caused it may be remotely triggered, such as an incoming L2CAP or RFCOMM
    connect request. To track this information introduce a new hci_conn flag
    called HCI_CONN_AUTH_INITIATOR.
    
    Signed-off-by: Johan Hedberg <johan.hedberg@intel.com>
    Signed-off-by: Marcel Holtmann <marcel@holtmann.org>

diff --git a/include/net/bluetooth/hci_core.h b/include/net/bluetooth/hci_core.h
index eb2b9c9e3480..996ed065b6c2 100644
--- a/include/net/bluetooth/hci_core.h
+++ b/include/net/bluetooth/hci_core.h
@@ -547,6 +547,7 @@ enum {
 	HCI_CONN_SECURE,
 	HCI_CONN_FIPS,
 	HCI_CONN_STK_ENCRYPT,
+	HCI_CONN_AUTH_INITIATOR,
 };
 
 static inline bool hci_conn_ssp_enabled(struct hci_conn *conn)